src/bun.js/bindings/ZigGlobalObject.cpp

```diff
@@ -2284,6 +2284,18 @@ extern "C" void ReadableStream__cancel(JSC__JSValue possibleReadableStream, Zig:
     ReadableStream::cancel(*globalObject, readableStream, exception);
 }
 
+extern "C" void ReadableStream__tee(JSC__JSValue possibleReadableStream, Zig::GlobalObject* globalObject, JSC__JSValue ptr[2])
+{
+    auto* readableStream = jsDynamicCast<JSReadableStream*>(JSC::JSValue::decode(possibleReadableStream));
+    if (UNLIKELY(!readableStream))
```

Comment:
just wondering, is there a specific reason why we don't use LIKELY(readableStream)?

Reply:
this condition is likely to be true: `!readableStream`

This path only happens if you call this with a |this| value that is not JSReadableStream.